registry/storage/driver/azure/azure.go

```diff
@@ -118,7 +119,7 @@ func (d *driver) GetContent(ctx context.Context, path string) ([]byte, error) {
 
 // PutContent stores the []byte content at a location designated by "path".
 func (d *driver) PutContent(ctx context.Context, path string, contents []byte) error {
-	if limit := 64 * 1024 * 1024; len(contents) > limit { // max size for block blobs uploaded via single "Put Blob"
+	if limit := 256 * 1024 * 1024; len(contents) > limit { // max size for block blobs uploaded via single "Put Blob"
```

Comment:
This limit was changed on the Azure side? Was this documented somewhere, I know this hard coded value has been confusing in the past

Reply:
Yes. The new SDK is using a newer version header when sending request. The new version supports 256MB. This will allow all driver tests to pass (previously there was one failed test due to the size limit here).
https://docs.microsoft.com/en-us/rest/api/storageservices/put-blob#remarks

Reply:
I updated the PR with comments.